modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -325,7 +326,13 @@
 					}
 				}else{
 					var width = this.getConfig("fixedControls") ? $( ".playlistInterface" ).width() - this.getConfig("horizontalControlsWidth")*2 : $( ".playlistInterface" ).width();
-					this.setConfig( 'mediaItemWidth', Math.floor(width / this.getConfig("MinClips")) );
+					// decide width of the items
+					// If should fill remaining horizontal space use preconfigured 'MinClips'
+					// else use cached initial 'MinClips' config
+					var itemsNumber = this.getConfig('horizontalFillRemainingSpace') ?
+						this.getConfig('MinClips') : this.minClips;
```

Comment:
@vladm3 @eitanavgil  - I don't understand the logic here. as far as I can tell this.getConfig('MinClips') and this.minClips return the same value.
@eitanavgil  - lets discuss over Skype